benchmarks/send-recv-core.py

```diff
@@ -290,13 +351,13 @@ def client(queue, port, server_address, args):
     med = format_bytes(2 * args.n_bytes / np.median(times))
     print(f"Average         | {avg}/s")
     print(f"Median          | {med}/s")
-    print("--------------------------")
-    print("Iterations")
-    print("--------------------------")
-    for i, t in enumerate(times):
-        ts = format_bytes(2 * args.n_bytes / t)
-        ts = (" " * (9 - len(ts))) + ts
-        print("%03d         |%s/s" % (i, ts))
+    # print("--------------------------")
+    # print("Iterations")
+    # print("--------------------------")
+    # for i, t in enumerate(times):
+    #     ts = format_bytes(2 * args.n_bytes / t)
+    #     ts = (" " * (9 - len(ts))) + ts
+    #     print("%03d         |%s/s" % (i, ts))
```

Comment:
Should we make this info optional?

Reply:
Good idea, added a new `--no-detailed-report` to suppress that.